about.html

```diff
@@ -0,0 +1 @@
+<p>about page</p>
```

Comment:
Hey, you can use react instead of plain html

Reply:
okay, we will address it in the next release